Added json response for multiple files

duongvy0112 opened the pull request:
- updated `unstructured_api_tools/pipelines/templates/pipeline_app.txt` to specify the api version
- updated `unstructured_api_tools/pipelines/templates/pipeline_api.txt` to add ability the json response for multiple files
- add new version `0.4.8` on `CHANGELOG.md`
- version sync in `unstructured_api_tools/__version__.py`
- passed `make test` and `make check`

cragwolfe commented:
@duongvy0112 , please include test instructions for the last 2 bullets in the [issue](https://github.com/Unstructured-IO/community/issues/36):

* Test instructions demonstrate json payloads being returned for a multi-file requests in a locally running pipeline-sec-filings API.
* Test instructions demonstrate json payloads being returned for a multi-file request in a locally running pipeline API that accepts files (in contrast to the text_files input in the sections API of pipeline-sec-filings).

Since we don't have great automated tested for this library yet, this needs to be a somewhat manual process.

For the first bullet, you can test in an pipeline-sec-filings environment with a `pip uninstall unstructured-api-tools` then from your unstructured-api-tools dir, do `pip install -e .`, then from the pipeline-sec-filings dir, do `make generate-api`. Please provide sample POST commands and descriptions of expected output. This can be in the PR description itself.

For the second bullet, you can create a new pipeline notebook in pipeline-sec-filings with a `pipeline_api(file, file_content_type, filename, ...)` definition that does something trivial like return the name of the file.

It would also be good to test the mixed case, for pipeline_api's that accept either file or text file:  `pipeline_api(text, file, file_content_type, filename, ...)`.

duongvy0112 commented:
- updated `unstructured_api_tools/pipelines/templates/pipeline_api.txt` to support mixed case input
- test with multiple files input
- test with mixed case input
- added bullet on this [issue](https://github.com/Unstructured-IO/community/issues/36)

duongvy0112 commented:
Fixed merge conflict
